src/st_state_patch.py

```diff
@@ -0,0 +1,219 @@
+"""Another prototype of the State implementation.
```

Comment:
You can ignore this file, it's a hack from `streamlit` until the properly implement this functionality